src/components/CampaignContactsChoiceForm.jsx

```diff
@@ -122,7 +122,6 @@ export class CampaignContactsChoiceFormBase extends React.Component {
               saveDisabled={this.props.saveDisabled}
               saveLabel={this.props.saveLabel}
               clientChoiceData={ingestMethod && ingestMethod.clientChoiceData}
-              lastResult={lastResult}
```

Comment:
There's no reason this should be deleted.  I thought it was an experiment, but it exists throughout the codebase and needs to come back to it's section.  I really wish that lastResult was the actual script.  Or a contact.  Restoring it is my top priority.